### PR TITLE
Updating homepages and sources from two BSC packages

### PR DESCRIPTION
**Pull request by kserradell**
Recently we changed bsc website so links were no longer valid.

I add to add a flag in binutils to avoid an error when using binutils in extrae.

KiM

### CODE REVIEW

File: var/spack/repos/builtin/packages/binutils/package.py [hunk 1]
```diff
@@ -76,6 +76,8 @@ def install(self, spec, prefix):
 
         if '+libiberty' in spec:
             configure_args.append('--enable-install-libiberty')
+            configure_args.append('--disable-nls')
```
Comment (davydden) [inline var/spack/repos/builtin/packages/binutils/package.py]: @jppelteret i think this fixes https://github.com/LLNL/spack/issues/3213

Comment (jppelteret) [inline var/spack/repos/builtin/packages/binutils/package.py]: Cool, thanks @davydden !

File: var/spack/repos/builtin/packages/binutils/package.py [hunk 2]
```diff
@@ -76,6 +76,8 @@ def install(self, spec, prefix):
...
+
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/binutils/package.py]: This extra line will cause the flake8 tests to fail.

**Comment by adamjstewart** [comment]
@kserradell Is there anywhere we can retrieve the older versions from? Like an archive directory?

File: var/spack/repos/builtin/packages/binutils/package.py (hunk 1 above)
Comment (jppelteret) [inline var/spack/repos/builtin/packages/binutils/package.py]: Well, not quite actually. The `libiberty` variant is not built by default, so this would in fact have no effect when the default build settings are used.

File: var/spack/repos/builtin/packages/binutils/package.py (hunk 2 above)
Comment (kserradell) [inline var/spack/repos/builtin/packages/binutils/package.py]: @adamjstewart how can this be fixed? Sorry for my lack of knowledge.

**Comment by kserradell** [comment]
@adamjstewart let me check with BSC developers.

File: var/spack/repos/builtin/packages/binutils/package.py (hunk 2 above)
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/binutils/package.py]: Just delete the blank line you added. According to PEP-8, you should only have a single blank line inside of a class. Outside of a class you should have exactly 2. Flake8 checks for PEP-8 compliance.

File: var/spack/repos/builtin/packages/binutils/package.py (hunk 1 above)
Comment (jppelteret) [inline var/spack/repos/builtin/packages/binutils/package.py]: But I can confirm that it does fix the build problem I'm getting on `MacOS 10.11` (#3213) if its moved to the main set of configuration arguments:
```
    def install(self, spec, prefix):
        configure_args = [
            '--prefix=%s' % prefix,
            '--disable-dependency-tracking',
            '--disable-werror',
            '--enable-interwork',
            '--enable-multilib',
            '--enable-shared',
            '--enable-64-bit-bfd',
            '--enable-targets=all',
            '--disable-nls',
            '--with-sysroot=/']
```

Can we put this option there instead?

**Comment by kserradell** [comment]
@adamjstewart 
extrae has only 3.4.1 version in the ftp
paraver has 4.6.2 and 4.6.3 so we could keep both versions

File: var/spack/repos/builtin/packages/binutils/package.py (hunk 1 above)
Comment (kserradell) [inline var/spack/repos/builtin/packages/binutils/package.py]: @jppelteret I can push it this PR if you agree.

Comment (adamjstewart) [inline var/spack/repos/builtin/packages/binutils/package.py]: Honestly, these changes are completely unrelated to your homepage changes. I would create a separate PR for any binutils changes.

Comment (kserradell) [inline var/spack/repos/builtin/packages/binutils/package.py]: Extrae was not building without this option (in my openSuse) but I have no problem to put it in a separate PR.

Comment (adamjstewart) [inline var/spack/repos/builtin/packages/binutils/package.py]: Thanks! I think it will be much easier to get this merged if it's two separate PRs. The homepage changes are pretty easy to approve. Changes to binutils may take longer to approve.

Comment (jppelteret) [inline var/spack/repos/builtin/packages/binutils/package.py]: Thanks @kserradell that would be great! I especially appreciate it since you'd have to go to the effort of creating another PR :-)

Comment (kserradell) [inline var/spack/repos/builtin/packages/binutils/package.py]: I did it #3218.

**Comment by viniciusvgp** [comment]
#7940
